eng/helix/content/runtests.cmd

```diff
@@ -23,14 +23,20 @@ echo "Installing SDK"
 powershell.exe -NoProfile -ExecutionPolicy unrestricted -Command "[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12; &([scriptblock]::Create((Invoke-WebRequest -useb 'https://dot.net/v1/dotnet-install.ps1'))) -Architecture %$arch% -Version %$sdkVersion% -InstallDir %DOTNET_ROOT%"
 echo "Installing Runtime"
 powershell.exe -NoProfile -ExecutionPolicy unrestricted -Command "[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12; &([scriptblock]::Create((Invoke-WebRequest -useb 'https://dot.net/v1/dotnet-install.ps1'))) -Architecture %$arch% -Runtime dotnet -Version %$runtimeVersion% -InstallDir %DOTNET_ROOT%"
+
+REM "Rename default.NuGet.Config to NuGet.Config if there is not a custom one from the project"
+if not EXIST ".\NuGet.Config" (
+    copy default.NuGet.Config NuGet.Config
+)
+
 echo "Checking for Microsoft.AspNetCore.App"
 if EXIST ".\Microsoft.AspNetCore.App" (
     echo "Found Microsoft.AspNetCore.App, copying to %DOTNET_ROOT%\shared\Microsoft.AspNetCore.App\%runtimeVersion%"
     xcopy /i /y ".\Microsoft.AspNetCore.App" %DOTNET_ROOT%\shared\Microsoft.AspNetCore.App\%runtimeVersion%\
 
     echo "Adding current directory to nuget sources: %HELIX_WORKITEM_ROOT%"
-    dotnet nuget add source %HELIX_WORKITEM_ROOT%
-    dotnet nuget add source https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet5/nuget/v3/index.json
+    dotnet nuget add source %HELIX_WORKITEM_ROOT% --configfile NuGet.Config
```

Comment:
@HaoK FYI, this was modifying global machine state and was sometimes causing failures on future test runs.

Reply:
Ah yeah that's not good.  Is there any way we can scope the sources to only a specific execution context (i.e. our runtests)?